File: Practice/21/C++/21. Индекс массы тела/21. Индекс массы тела/21. Индекс массы тела.cpp

#include<iostream>

double BMI(double weight, double height)
{
	return weight / pow(height, 2);
}
void printBMI(double BMI)
{
	if (BMI < 18.5)
		std::cout << "Underweight"; if (BMI >= 18.5 && BMI < 25.0) std::cout << "Normal";
	if (BMI >= 25.0 && BMI < 30) std::cout << "Overweight"; if (BMI >= 30.0) std::cout << "Obesity";
}

int main()
{
	std::cout << "weight && height: ";
	double weight,height;
	std::cin >> weight >> height;
	height /= 100;
	printBMI(BMI(weight, height));

}

File: Practice/21/Python/21. Индекс массы тела/21. Индекс массы тела/_21._Индекс_массы_тела.py
weight, height = map(int, input("weight and height: ").split())
height /= 100
def bmi(weight: float, height: float) -> float:
    return weight / (height*height)
def print_bmi(bmi: float) -> float:
    if (bmi < 18.0):
        print("Underweight")
    if (bmi >= 18.0 and bmi < 25.0):
        print("Normal")
    if (bmi >= 25.0 and bmi < 30.0):
         print("Overweight")
    if (bmi > 30):
        print("Obesity")
print_bmi(bmi(weight, height))
